services/order.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from db.models import Order, Ticket, MovieSession
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+
+    with transaction.atomic():

```

Comment:
use @transaction.atomic() decoreator

Reply:
With decorator it will work the same, but pros is in less rows of code ?